doc/sphinx/conf.py.in

```diff
@@ -180,6 +180,7 @@ man_pages = [
     ("docs/fpga_tools/opaevfio/opaevfio", 'opaevfio', u'Bind/Unbind accelerator to/from vfio-pci', [author], 8),
     ("docs/fpga_tools/opaeuio/opaeuio", 'opaeuio', u'Bind/Unbind DFL device to/from dfl-uio-pdev', [author], 8),
     ("docs/fpga_tools/userclk/userclk", 'userclk', u'Set AFU high and low clock frequency', [author], 8),
+    ("docs/fpga_tools/host_exerciser.md/host_exerciser", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
```

Comment:
I think that you have host_exerciser.md/host_exerciser in reverse order.  ie, shouldn't it be
docs/fpga_tools/host_exerciser/host_exerciser.md ?

Reply:
updated with proper HE md file  path docs/fpga_tools/host_exerciser/host_exerciser.md